File: Exercises_Python/1131.py


saida = 0
soma_gols_inter = 0
soma_gols_gre = 0
vitorias_inter = 0
vitorias_gre = 0
empate = 0
count = 0
while saida != 2:
    gols_inter,gols_gre = map(int, input().split())

    if gols_inter > gols_gre:
        vitorias_inter += 1
    elif gols_gre > gols_inter:
        vitorias_gre += 1
    else:
        empate += 1

    soma_gols_inter += gols_inter
    soma_gols_gre += gols_gre
    count += 1
    
    print('Novo grenal (1-sim 2-nao)')
    saida = int(input())


print('{} grenais'.format(count))
print('Inter:{}'.format(vitorias_inter))
print('Gremio:{}'.format(vitorias_gre))
print('Empates:{}'.format(empate))
if vitorias_inter > vitorias_gre:
    print('Inter venceu mais')
elif vitorias_gre > vitorias_inter:
    print('Gremio venceu mais')
else:
    print('Nao houve vencedor')

File: Bee-Crowd/Exercises_C++/1131.cpp
#include <iostream>
using namespace std;

int main() {

    int gols_gremio,gols_inter,count,ler_de_novo,v_inter,v_grem,i,g,empates;
    count = 0;
    ler_de_novo = 1;
    v_inter = 0;
    v_grem = 0;
    empates = 0;

    while (ler_de_novo == 1){
        cin >> i >> g;

        if (i > g) {
            v_inter += 1;
        }
        else if (i < g) {
            v_grem += 1;
        }
        else if (i == g){
            empates += 1;
        }

        count += 1;
        cout << "Novo grenal (1-sim 2-nao)" << endl;
        cin >> ler_de_novo;
    }

    cout << count << " grenais" << endl;
    cout << "Inter:" << v_inter << endl;
    cout << "Gremio:" << v_grem << endl;
    cout << "Empates:" << empates << endl;
    
    if (v_inter > v_grem) {
        cout << "Inter venceu mais" << endl;
    }
    else if (v_inter < v_grem) {
        cout << "Gremio venceu mais" << endl;
    }
    else {
        cout << "Nao houve vencedor" << endl;
    }

}